worker/worker.go

```diff
@@ -331,6 +333,16 @@ func (w *Worker) ScheduleJobCompletedJob(jobID string, at int64) (string, error)
 // Start starts the worker
 func (w *Worker) Start() {
 	jobsStatsPort := w.Config.GetInt("workers.statsPort")
-	go workers.StatsServer(jobsStatsPort)
-	workers.Run()
+	http.HandleFunc("/stats", func(w http.ResponseWriter, req *http.Request) {
```

Comment:
I understand the idea of always returning success here, but the worker might say it is healthy in situations it should probably die using this approach. A middle ground approach that actually checked the status of the worker would probably be better.

Reply:
I agree with you, by the worker lib that we are using for workers don't expose any information about the internal state of the workers.

Reply:
There is a part of the validation that is:
```	conn := Config.Pool.Get()
	defer conn.Close()

	conn.Send("multi")
	conn.Send("get", Config.Namespace+"stat:processed")
	conn.Send("get", Config.Namespace+"stat:failed")
	conn.Send("zcard", Config.Namespace+RETRY_KEY)
```
You could do something similar to ensure connectivity with redis, doing a simple get